tests/test_image_to_text_example.py

```diff
@@ -73,7 +77,8 @@ def _test_image_to_text(
         ]
 
     command.append("--bf16")
-    command.append("--sdp_on_bf16")
```

Comment:
@mengker33 Could you please explain the reason for removing this option from the `THUDM/glm-4v-9b` model?

Reply:
I have tested the example in the README file and also the test using `--sdp_on_bf16`, but I did not observe any noticeable difference.

Reply:
As I remembered, I met the error with `--sdp_on_bf16`, something like `torch._C._set_math_sdp_allow_fp16_bf16_reduction` not supported... Most probably I was using 1.18 machine when adding this part and no errors showed on 1.19 machine now. 
I didn't see differences either with or without setting `--sdp_on_bf16`. But to align with other test model, let me add it back and update the throughput baseline. 
Really appreciate your comment!